README.md

```diff
@@ -46,6 +46,49 @@ We take privacy seriously, and do not perform any automated error collection. In
 Thank you kindly!
 ```
 
+The error dump file generated by `human-panic` contains the following fields.
+
+```norun
```

Comment:
rustdoc assumes code to be `rust` (makes sense), so instead of `norun` you should probably set it to `toml` which would be more correct and more highlighted on github